docker/scripts/install-docker-compose-ubuntu.sh

```diff
@@ -15,10 +21,13 @@ sudo true
 wget -qO- https://get.docker.com/ | sh
 
 # Install docker-compose
-COMPOSE_VERSION=`git ls-remote https://github.com/docker/compose | grep refs/tags | grep -oE "[0-9]+\.[0-9][0-9]+\.[0-9]+$" | sort --version-sort | tail -n 1`
-sudo sh -c "curl -L https://github.com/docker/compose/releases/download/${COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose"
-sudo chmod +x /usr/local/bin/docker-compose
-sudo sh -c "curl -L https://raw.githubusercontent.com/docker/compose/${COMPOSE_VERSION}/contrib/completion/bash/docker-compose > /etc/bash_completion.d/docker-compose"
+readonly dockerComposeVersion='1.21.0'
+
+curl -Ls "https://github.com/docker/compose/releases/download/$dockerComposeVersion/docker-compose-$(uname -s)-$(uname -m)" \
+    -o /usr/local/bin/docker-compose
```

Comment:
In the case of docker-compose there was no way to install from distro packages (at least for CentOS), that's why we want for vendor packages.

Reply:
I can understand.